client/src/lib.rs

```diff
@@ -0,0 +1,133 @@
+/// A temporary module that will be replaced by the real one
+/// exported by the Simperby main repository.
+pub mod common;
+
+use common::*;
+use rust_decimal::prelude::*;
+use serde::{Deserialize, Serialize};
+use std::collections::HashMap;
+use thiserror::Error;
+/// Information of a contract.
+#[derive(Serialize, Deserialize, Debug, PartialEq, Eq, Clone)]
+pub struct ContractInfo {
+    /// The address of the contract.
+    pub address: String,
+    /// The increasing sequence that is for preventing the replay attack.
+    ///
+    /// A valid message from PBC MUST carry the same number with this,
+    /// in order to succesfully convince the contract. (i.e., this number is something that the consensus should have finalized on-chain).
+    ///
+    /// - Note1: this is totally irrelevant to the account sequence.
+    /// - Note2: the light client contract doesn't need this because the 'block height' provides the same safety guard.
```

Comment:
Do we need this Note2? We decided to combine treasury & light client contracts.

Reply:
fixed